client/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:8.15.0-alpine
```

Comment:
Do we want to place any constraint around the base image used e.g. are would we prefer people to keep alpine and add what they need or are we happy for people to put any old base image in the dockerfiles?

Reply:
I'm not sure. I prefer alpine and add because it exposes some idea of what the candidate is thinking about, whereas if they just change to pull the standard image it is harder for us to figure out what exactly they're using from the image.

Reply:
Having thought more I think I'd vote to constrain the base image to the alpine one provided. It won't hamper anyone unfamiliar with docker as they have enough to run the provided code. Anyone choosing to tinker can do so but must make it explicit to us what they are changing. It also avoids any concerns about dodgy base images being swapped in.